mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -116,10 +122,22 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
   };
 
   /**
-   * Update value
+   * Check is value valid or not according min length
```

Comment:
Wrong comment, function not only  check value

Reply:
done